aws-load-balancer-controller.yaml

```diff
@@ -20,7 +20,7 @@ pipeline:
     with:
       repository: https://github.com/kubernetes-sigs/aws-load-balancer-controller
       tag: v${{package.version}}
-      expected-commit: 06abaed66e17a411ba064f34e6018b889780ac66
+      expected-commit: fe6f4c00f9cb8e996655c85493f58efe8a9d4082
```

Comment:
Merging failed with

```
ℹ️  x86_64    | Error (git-checkout): expect commit fe6f4c00f9cb8e996655c85493f58efe8a9d4082, got ba7df31f59d44e0f18d50015e6ae3f6f1beb6fdf
```

Looking into this.

Reply:
https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/tag/v2.5.2 says v2.5.2 is ba7df31f59d44e0f18d50015e6ae3f6f1beb6fdf -- I'm not sure how wolfibot got fe6f4c 🤔 

@rawlingsj

Reply:
I just tried rerunning the updater on [my fork](https://github.com/patflynn/os/pull/147/files) and it picked up my the right commit. Going to close and hope this fixes itself.